tools/osx/crosstool/BUILD.toolchains

```diff
@@ -19,5 +39,6 @@ OSX_DEVELOPER_PLATFORM_CPUS = [
         toolchain = "@local_config_cc//:cc-compiler-" + arch,
         toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
     )
-    for arch in OSX_TOOLS_ARCHS for cpu in OSX_DEVELOPER_PLATFORM_CPUS
+    for arch in OSX_TOOLS_ARCHS + ["armeabi-v7a"]
```

Comment:
we add this manually here so we can exclude it from the `OSX_TOOLS_ARCHS` so it doesn't get an autocreated toolchain in the other loops, but we still need to reference the manually created one here, so we keep it in the `OSX_TOOLS_CONSTRAINTS` dictionary

Reply:
Yep. Without this you get this error when running android tests https://buildkite.com/bazel/bazel-bazel-github-presubmit/builds/8780#9ad3b275-6f59-48f9-b0d7-38c4a6793694